### PR TITLE
O365 uplift

### PR DESCRIPTION
**Pull request by oobedan**
Updates include (O365 abac and design):
- Overall some typos and grammar
- Update for DKIM, SPF & DMARC policy 
- DLP alignment and updates support for sensitive info types, device DLP
- M365 group creation lockdown updates (included update to platform design)
- Guidance around retention vs backup

### CODE REVIEW

File: blueprint/office-365.md [hunk 1]
```diff
@@ -118,59 +118,63 @@ Show the users name on the top navigation bar when the user is signed in | Yes |
...
+* Power Platform administrator.
+* Customer Lockbox Administrator.
+* Intune administrator.
+* Office Apps administrator.
+* Message Center Privacy Reader.
+* Message Center Reader.
+* Security administrator.
+* Security reader.
+
+Note, using PIM for the SharePoint administrator role, the Device administrator role, and roles trying to access the Microsoft Security and Compliance Center might experience delays of up to a few hours after activating the role, see [](https://docs.microsoft.com/en-us/azure/active-directory/privileged-identity-management/pim-roles) for further information.
```
Comment (zedzedzed) [inline blueprint/office-365.md:174]: Is this the right link about the delay? It sends me to a page titled: `Roles you can't manage in Privileged Identity Management`

File: blueprint/office-365.md [hunk 2]
```diff
@@ -260,29 +265,32 @@ Add regional egress points | Not Configured | All traffic will egress centrally
...
+This section is only relevant for agencies implementing a hybrid solution that leverages an on-premises Exchange Server(s).
 
-A Hybrid configuration provides administrators with added flexibility to transition users to the Cloud without isolating them from the on-premises resources. 
+Exchange Online can be used standalone (cloud only) or integrated with an on-premises Exchange Server(s) and Active Directory Domain Services, extending the organisations messaging farm in a hybrid configuration.
 
-This section is only relevant for agencies implementing a hybrid solution that leverages an on-premises Exchange Server(s)
+A Hybrid configuration provides administrators with added flexibility to transition users to the Cloud without isolating them from the on-premises resources. Hybrid with can also assist with transport routing for compliance reasons (e.g. Govlink) when Hybrid with "centralized mail transport" enabled. The [Edge Transport](https://docs.microsoft.com/en-us/exchange/edge-transport-servers) service may be deployed in scenarios where the agency does not wish to expose Hybrid mail servers directly to Exchange Online Protection.
```
Comment (zedzedzed) [inline blueprint/office-365.md:284]: I think this sentence is incomplete. `Hybrid with` what?

```Hybrid with can also assist with transport routing for compliance reasons (e.g. Govlink) when Hybrid with "centralized mail transport" enabled.```

File: blueprint/office-365.md (hunk 1 above)
Comment (oobedan) [reply blueprint/office-365.md:174]: Yeah its correct, if you scroll down the page it is mentioned in a <Note!> block warning about the issue with the delay using PIM for various roles.

File: blueprint/office-365.md (hunk 2 above)
Comment (oobedan) [reply blueprint/office-365.md]: have reworded to - 'when "centralized mail transport" is enabled.

Comment (zedzedzed) [reply blueprint/office-365.md]: Sorry I should have been more clearer. I meant the first `Hybrid with` at the start of the sentence:

`Hybrid with can also assist with transport`

Comment (oobedan) [reply blueprint/office-365.md]: @zedzedzed sorry missed that one. Updated to "A Hybrid configuration".

**Review by zedzedzed — APPROVED** [review]

**Pull request merged**